Sources/Testing/Testing.swift

```diff
@@ -0,0 +1,25 @@
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2024 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See http://swift.org/LICENSE.txt for license information
+// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+
+#if canImport(Glibc)
+import Glibc
+#elseif canImport(Musl)
+import Musl
+#elseif os(WASI)
+import WASILibc
+#elseif canImport(CRT)
```

Comment:
nit I think we should might as well add `#if canImport(Bionic)` here as well. But def none blocking

Reply:
Fair point - I'm not sure if we've tried running SwiftPM tests on android for this repo yet so I'll follow up with that later once we try to do that